src/main/java/com/matera/cadastrocentral/telephone/TelephoneService.java

```diff
@@ -0,0 +1,110 @@
+package com.matera.cadastrocentral.telephone;
+
+import javassist.tools.web.BadHttpRequest;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+@Service
+public class TelephoneService {
+
+    @Autowired
+    public TelephoneService(TelephoneRepository telephoneRepository) {
+        this.telephoneRepository = telephoneRepository;
+    }
+
+    private final TelephoneRepository telephoneRepository;
+
+    public List<Telephone> findAllTelephonesByClientId(UUID clientId) {
+        List<Telephone> phoneList = telephoneRepository.findAllByClientId(clientId);
+        if (phoneList.isEmpty()) {
+            throw new ClientNotFound();
```

Comment:
it's ok to return a empty list if we don't find any telephones for the client. no exception needed here